toolchain/check/handle_function.cpp

```diff
@@ -489,40 +489,45 @@ static auto BuildFunctionDecl(Context& context,
   return {function_decl.function_id, decl_id};
 }
 
-CARBON_DIAGNOSTIC(
-    UnusedMarkerInDeclaration, Error,
-    "`unused` marker cannot be used on a declaration that is not a "
-    "definition");
-
+// Checks that "unused" marker does not appear in declarations that
+// are not definitions and emit a diagnostic for every binding that
+// is marked unused.
+// We do this by looking at all name bindings, with special treatment
```

Comment:
We use blank lines to separate paragraphs in comments.
```suggestion
//
// We do this by looking at all name bindings, with special treatment
```

Reply:
Moved the comment down.